configuration/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.4"
+
+services:
+  etcd1:
+    image: gcr.io/etcd-development/etcd:v3.3
+    restart: always
```

Comment:
I personally prefer 'unless-stopped' to 'always', but thats simple a suggestion.

Reply:
As long as they both deal with the reboot case

Reply:
nO JOeL iTs HIgH AvAILaBIlITy!!!!@!@!@#!@#

Reply:
> The unless-stopped restart policy behaves the same as always with one exception. When a container is stopped and the server is rebooted or the Docker service is restarted, the container will not be restarted.

Reply:
I'm always of the view that if an admin/user/somebody has taken the decision to stop a container, that it should remain stopped until explicitly started again.
Honestly though I don't think it matters that much and if we change our minds later it's a trivial change to make.